ios/Podfile.lock

```diff
@@ -418,6 +418,8 @@ PODS:
   - urbanairship-react-native (10.0.0):
     - Airship (= 14.2.0)
     - React-Core
+  - RNSVG (12.1.0):
```

Comment:
Is this from a library of react?
From Xcode 12 onwards iOS supports svgs, so we souldn't need any pods added to the podfile.

Reply:
Yeah, I believe this is the iOS bridge for `react-native-svg`